python/vyos/ifconfig/interface.py

```diff
@@ -1656,6 +1677,11 @@ def update(self, config):
             for addr in tmp:
                 self.add_ipv6_eui64_address(addr)
 
+        # Configure IPv6 base time in milliseconds - has default value
+        tmp = dict_search('ipv6.base_reachable_time', config)
+        value = tmp if (tmp != None) else '30'
```

Comment:
The else path is not required as there is an interface default value. Please drop it in a subsequent commit